source/view/videoController.js

```diff
@@ -15,63 +15,65 @@
 var videos = new Array();
 var videoID = 0;
 var playingVideo;
+var mediaManagerID;
+var videoManagerID;
 
-function loadVideo( src, type, dontRemoveWhenEnded ) {
+function loadVideo( src, type ) {
     var video = {
         "id" : videoID,
-        "elem" : document.createElement( "video" ),
-        "source" : document.createElement( "source" ),
-        "wrapper" : document.createElement( "div" )
     };
-    video.elem.appendChild( video.source );
-    video.wrapper.appendChild( video.elem );
-    video.source.src = "assets/video/" + src;
-    video.source.type = type || 'video/mp4;codecs="avc1.42E01E,mp4a.40.2"';
-    video.elem.id = "video" + video.id;
-    video.elem.className = "video";
-    video.wrapper.className = "videoWrapper";
-    video.elem.load();
+    video.source = "assets/video/" + src;
+    video.vidName = src;
 
     document.onkeypress = removeVideoOnEvent;
 
-    if ( !dontRemoveWhenEnded ) {
-        video.elem.onended = removeVideoOnEvent;
-    }
-
+    mediaManagerID = vwf_view.kernel.find( undefined, "/mediaManager" )[ 0 ];
+    videoManagerID = vwf_view.kernel.find( mediaManagerID, "videoManager" ) [ 0 ];
     videos.push( video );
     videoID++;
     return videoID - 1;
+
+    var rover = vwf_view.kernel.find( "", "//rover" )[ 0 ];
 }
 
 function playVideo( id ) {
     var video = videos[ id ];
     if ( video ) {
-        document.body.appendChild( video.wrapper );
+
         playingVideo = video;
-        video.elem.play();
+        vwf_view.kernel.setProperty( videoManagerID, "z_index", 103 );
+        vwf_view.kernel.callMethod( videoManagerID, "show" );
```

Comment:
Since `z_index` is a property, we should probably just add the property setting to the `show` method and make a single model call.